crates/sui-core/src/authority/execution_time_estimator.rs

```diff
@@ -243,6 +248,13 @@ impl ExecutionTimeObserver {
                         .epoch_execution_time_observer_overutilized_objects
                         .dec();
                 }
+                if self.config.report_object_utilization_metric() && utilization.was_overutilized {
+                    epoch_store
+                        .metrics
+                        .epoch_execution_time_observer_object_utilization
+                        .with_label_values(&[id.to_string().as_str()])
+                        .set(utilization.excess_execution_time.as_secs_f64());
```

Comment:
shouldn't we just use a counter and add the last observed execution time (if the object was overutilized)? then we can use `rate()` queries

Reply:
I'm not sure I follow, why is that better?

I figure we would just plot this directly (or compute percentiles over all the labels) and then that's exactly what we'd want to see, no?

Reply:
Nevermind, I think I see what you're saying